CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.0
```

Comment:
I've been isolating the `CHANGELOG` changes to their own commit (usually with the commit message of a version number). Can you remove this changeset so that we can apply it during the tagging step?